test/JDBC/expected/BABEL-3248.out

```diff
@@ -65,6 +65,11 @@ Query Text: select 1
 Result
 ~~END~~
 
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.063 ms
```

Comment:
Did we still need to print the actual time when : 
select set_config('babelfishpg_tsql.explain_summary', 'off', false); 
?

Reply:
See the def of this variable : 
A boolean that turns on (or off) summary information such as total time after the query plan. (Default: on) (Allowable: off, on)

Reply:
Updated logic to show the parsing time only when `babelfishpg_tsql.explain_summary` is enabled